src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const styles = sourceString.split(';');
```

Comment:
[CODE KNOWLEDGE] - There's no need to make a copy before each operation. The `split` method does not mutate the original string, so you can chain the `forEach` directly without assigning it to the `styles` variable.